src/umpire/util/MPI.cpp

```diff
@@ -0,0 +1,62 @@
+#include "umpire/config.hpp"
+
+#include "umpire/util/MPI.hpp" 
+
+#include "umpire/util/Macros.hpp"
+
+#if defined(UMPIRE_ENABLE_MPI)
+#include "mpi.h"
+#endif
+
+namespace umpire {
+namespace util {
+
+int MPI::s_rank = -1;
+int MPI::s_world_size = -1;
+bool MPI::s_initialized = false;
+
+void 
+MPI::initialize()
+{
+  if (!s_initialized) {
+#if !defined(UMPIRE_ENABLE_MPI)
```

Comment:
MPI support here!

Reply:
Does this build without including MPI?  Or are we always going to build with MPI?

Reply:
Of course. If MPI isn't enabled, we "fake" it and return 0 for rank, and 1 for size.